ui/apps/dashboard/src/components/Navigation/old/NavItem.tsx

```diff
@@ -36,6 +36,7 @@ export default function NavItem<PassedHref extends string>({
     <Link
       key={href.toString()}
       href={href}
+      prefetch={true}
```

Comment:
We prefetch our top level nav items. This doesn't buy us a ton as all of our page content fetches data client side. But this does allow us to send down our pre-rendered nav and breadcrumbs.